ui/src/containers/App.js

```diff
@@ -34,7 +34,6 @@ class App extends Component {
 
   render() {
     const { language, api, theme } = this.props.config;
-
```

Comment:
minor: this seems totally unrelated to the current PR (the file is not touched except for this change).

It's nice to do cosmetics changes like that, but they deserve to be in a dedicated commit.

Don't bother splitting it this time, but please keep it in mind for a future PR.

Reply:
> minor: this seems totally unrelated to the current PR (the file is not touched except for this change).
> 
> It's nice to do cosmetics changes like that, but they deserve to be in a dedicated commit.
> 
> Don't bother splitting it this time, but please keep it in mind for a future PR.

I'll keep in mind.  Thanks for the kind advice!